configurations/forwardNew.yml

```diff
@@ -0,0 +1,19 @@
+features:
```

Comment:
Filename should explicitly include `hcal` and not refer to "new" (this is going to be the default when we merge this, and there's nothing new about it).

Reply:
no this is the whole forward setup, not just the hcal